frontend/src/lib/components/Alerts/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const INSIGHT_ALERT_FIRING_SUB_TEMPLATE_ID = 'insight-alert-firing'
+export const INSIGHT_ALERT_DESTINATION_LOGIC_KEY = 'insightAlertDestination'
+export const INSIGHT_ALERT_FIRING_EVENT_ID = '$insight_alert_firing'
```

Comment:
Can you move these constants into this file instead? https://github.com/PostHog/posthog/blob/master/frontend/src/lib/constants.tsx

Reply:
Thanks for helping me move :)